simple-git-hooks.js

```diff
@@ -296,14 +300,18 @@ function _getConfigFromFile(projectRootPath, fileName) {
 }
 
 /**
- * Validates the config, checks that every git hook is named correctly
+ * Validates the config, checks that every git hook or option is named correctly
  * @return {boolean}
- * @param {{string: string}} hooks
+ * @param {{string: string}} config
  */
-function _validateHooks(hooks) {
+function _validateHooks(config) {
+
+    for (let hookOrOption in config) {
+        if (!Object.prototype.hasOwnProperty.call(config, hookOrOption)) {
```

Comment:
I think we do not need to `continue` in this cycle:

We simply iterate over every entry (hookOrOption) in config, 
If enrty is bad, we raise an error (return false),

Reply:
@toplenboren Done!